src/styles/each.scss

```
@@ -0,0 +1,16 @@
+$week-days: (
+  "mon": 0, 
+  "tue": calc(#{$calendar-width} - #{$square-side} * 6 - #{$gap} * 5), 
+  "wed": calc(#{$calendar-width} - #{$square-side} * 5 - #{$gap} * 4),
+  "thu": calc(#{$calendar-width} - #{$square-side} * 4 - #{$gap} * 3),
+  "fri": calc(#{$calendar-width} - #{$square-side} * 3 - #{$gap} * 2),
+  "sat": calc(#{$calendar-width} - #{$square-side} * 2 - #{$gap}),
+  "sun": calc(#{$calendar-width} - #{$square-side}));
```

You can calculate the margin only once inside the loop. No need to repeat the same logic here 
```suggestion
$week-days: (
  "mon": 0, 
  "tue": 1, 
  "wed": 2,
  "thu": 3,
  "fri": 4,
  "sat": 5,
  "sun": 6;
```